.travis.yml

```diff
@@ -29,7 +29,7 @@ before_install:
 install:
   - travis_wait npm install
```

Comment:
Maybe switch this over to [`npm ci`](https://docs.npmjs.com/cli/ci.html) while we're at it

Reply:
sure did that, unfortunately`lerna bootstrap` won't use `npm ci` until we upgrade lerna to `3.x` we had problems with upgrading however, we should look into it again.

Reply:
lerna publish in the latest version fails with the current way we publish. Either we would need a tag in npm to use the current publish method or swap our current publish method to not do a canary release.

Reply:
or we can release a version, that would work too